freqtrade/freqtradebot.py

```diff
@@ -488,13 +482,30 @@ def execute_buy(self, pair: str, stake_amount: float, price: Optional[float] = N
 
         min_stake_amount = self.exchange.get_min_pair_stake_amount(pair, buy_limit_requested,
                                                                    self.strategy.stoploss)
+
+        if not self.edge:
```

Comment:
yeah edge ... 
it's honestly losing more and more of it's standalone possibilities - as most of the things it did are now possible to the strategy ... (every time i see a condition like this - i'm reminded of that).